packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -800,7 +800,7 @@ const listItemsRenderer = (props) => {
 	const {
 		cc,
 		handlePlaceholderFocus,
-		initItemContainerRef: initUiItemContainerRef,
+		itemContainerRef: initUiItemContainerRef,
```

Comment:
It seems to be a bug fix not an implementation for a new feature. Am I correct?

Reply:
Yes. Curious why we could not find this bug until now. Anyway NewVirtualList will work after fixing this bug. So I added this patch to my PR.